### PR TITLE
v10.0 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]

File: source/about/mattermost-v10-changelog.md [hunk 1]
```diff
@@ -7,17 +7,95 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - Added Do not disturb and late timezone warnings to Direct Message posts.
+ - Added user statuses to the Group Members modal.
+ - Added labels for channel header and purpose in the right-hand side channel info view.
+ - Added pagination user interface to the ``BackstageList`` component.
+ - Made various improvements to code involving user preferences.
+ - Promoted GIF picker, custom groups and message priority out of Beta.
+ - Removed the **Pre-release features** section from **Settings > Advanced** due to lack of usage.
+
+#### Administration
+ - Added experimental support for Redis. This is not intended for production use yet.
```
Comment (agnivade) [inline source/about/mattermost-v10-changelog.md:47]: Let's remove this. I don't want to confuse customers yet. We will announce when we are ready to announce.

**Review by agnivade — APPROVED** [review]
Thanks

**Review by marianunez — APPROVED** [review]

File: source/about/mattermost-server-releases.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Mattermost releases a new server version on the 16th of each month in [binary fo
 
 | **Release** | **Released on** | **Support ends** |
 |:---|:---|:---|
+| v10.0 [Download](https://releases.mattermost.com/10.0.0/mattermost-10.0.0-linux-amd64.tar.gz) \| {ref}`Changelog <release-v10-0-major-release>` | 2024-09-16 | 2024-12-15 |
```
Comment (wiggin77) [inline source/about/mattermost-server-releases.md:22]: @amyblais  I think this is a typo.  Support ends in December?

Comment (amyblais) [reply source/about/mattermost-server-releases.md:22]: All non-ESR releases are supported for 3 months.

File: source/about/mattermost-v10-changelog.md [hunk 3]
```diff
@@ -7,17 +7,93 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - Apps Framework is deprecated for new installs. Please extend Mattermost using webhooks, slash commands, OAuth2 apps, and plugins.
+ - An Enterprise license will be required to use v2.0+ Playbooks functionality starting in v10. Team Edition and Professional deployments can continue to use Playbooks v1.x. Feature updates will only be added to Playbooks v2.0+.
+ - Renamed ``Channel Moderation`` to ``Advanced Access Control`` in the channel management section in the **System Console**.
+ - Renamed announcement banner feature to “system-wide notifications”.
+ - Renamed “Collapsed Reply Threads” to “Threaded Discussions” in the System Console.
+ - Renamed “System Roles” to “Delegated Granular Administration” in the System Console.
+ - Fully deprecated the ``/api/v4/image`` endpoint when the image proxy is disabled.
+ - Pre-packaged Calls plugin [v1.0.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.0.0). This includes breaking changes such as allowing calls in direct message channels only on unlicensed servers.
+ - Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, ``AdvancedLoggingConfig``, and ``pageSize`` query parameter.
```
Comment (amyblais) [inline source/about/mattermost-v10-changelog.md:25]:
```suggestion
 - Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, ``AdvancedLoggingConfig``, and ``pageSize`` configuration fields.
```

File: source/upgrade/important-upgrade-notes.rst [hunk 4]
```diff
@@ -12,12 +12,34 @@ Important Upgrade Notes
...
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Pre-packaged Calls plugin `v1.0.0 <https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.0.0>`__. This includes breaking changes such as allowing|
+|                                                    | calls in direct message channels only on unlicensed servers.                                                                                                     |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, ``AdvancedLoggingConfig``, and ``pageSize`` query parameter.                              |
```
Comment (amyblais) [inline source/upgrade/important-upgrade-notes.rst:41]:
```suggestion
|                                                    | Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, ``AdvancedLoggingConfig``, and ``pageSize`` configuration fields.                              |
```

**Review by wiggin77 — APPROVED** [review]

File: source/about/deprecated-features.rst [hunk 5]
```diff
@@ -19,6 +19,15 @@ Mattermost Server v10.2.0
...
 ----------------------------------------
 
+Mattermost Server v10.0.0
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- We no longer support new installations using MySQL starting in v10. All new customers and/or deployments will only be supported with the minimum supported version of the PostgreSQL database. End of support for MySQL is targeted for Mattermost v11.
+- Apps Framework is deprecated for new installs. Please extend Mattermost using webhooks, slash commands, OAuth2 apps, and plugins.
+- ```
+- Fully deprecated the ``/api/v4/image`` endpoint when the image proxy is disabled.
+- Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, ``AdvancedLoggingConfig``, and ``pageSize`` configuration fields.
```
Comment (hanzei) [inline source/about/deprecated-features.rst:29]: `pageSize` is a query argument, not a config setting.

```suggestion
- Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, and ``AdvancedLoggingConfig`` configuration fields.
- Removed deprecated ``pageSize`` from most API endpoints.
```

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks @amyblais !

**Review by streamer45 — APPROVED** [review]

File: source/about/deprecated-features.rst [hunk 6]
```diff
@@ -19,6 +19,16 @@ Mattermost Server v10.2.0
...
 
+Mattermost Server v10.0.0
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- We no longer support new installations using MySQL starting in v10. All new customers and/or deployments will only be supported with the minimum supported version of the PostgreSQL database. End of support for MySQL is targeted for Mattermost v11.
+- Apps Framework is deprecated for new installs. Please extend Mattermost using webhooks, slash commands, OAuth2 apps, and plugins.
+- ```
+- Fully deprecated the ``/api/v4/image`` endpoint when the image proxy is disabled.
+- Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, and ``AdvancedLoggingConfig`` configuration fields.
+- Removed deprecated ``pageSize`` from most API endpoints.
```
Comment (hanzei) [inline source/about/deprecated-features.rst:30]:
```suggestion
- Removed deprecated ``pageSize`` query parameter from most API endpoints.
```

File: source/about/mattermost-v10-changelog.md [hunk 7]
```diff
@@ -7,17 +7,93 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - An Enterprise license will be required to use v2.0+ Playbooks functionality starting in v10. Team Edition and Professional deployments can continue to use Playbooks v1.x. Feature updates will only be added to Playbooks v2.0+.
+ - Renamed ``Channel Moderation`` to ``Advanced Access Control`` in the channel management section in the **System Console**.
+ - Renamed announcement banner feature to “system-wide notifications”.
+ - Renamed “Collapsed Reply Threads” to “Threaded Discussions” in the System Console.
+ - Renamed “System Roles” to “Delegated Granular Administration” in the System Console.
+ - Fully deprecated the ``/api/v4/image`` endpoint when the image proxy is disabled.
+ - Pre-packaged Calls plugin [v1.0.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.0.0). This includes breaking changes such as allowing calls in direct message channels only on unlicensed servers.
+ - Removed deprecated ``Config.ProductSettings``, ``LdapSettings.Trace``, and ``AdvancedLoggingConfig`` configuration fields.
+ - Removed deprecated ``pageSize`` from most API endpoints.
```
Comment (hanzei) [inline source/about/mattermost-v10-changelog.md:26]:
```suggestion
 - Removed deprecated ``pageSize`` query parameter from most API endpoints.
```

**Review by hanzei — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by cpoile — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**